modules/geocoder_geofield/src/Geocoder/Dumper/Geohash.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * @file
```

Comment:
Below @file we add:
Contains \Drupal\Full\Namespaced\Class\Name.

Note the dot at the end.